src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx

```diff
@@ -19,13 +20,77 @@ type HacbsApplicationDetailsProps = {
 const HacbsApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ applicationName }) => {
   const { namespace } = React.useContext(NamespaceContext);
   const navigate = useNavigate();
+  const { quickStarts } = useChrome();
+
   const [application, applicationsLoaded] = useK8sWatchResource<ApplicationKind>({
     groupVersionKind: ApplicationGroupVersionKind,
     name: applicationName,
     namespace,
   });
   const appDisplayName = application?.spec?.displayName || '';
 
+  useEffect(() => {
+    const hacbsShowAppQs = localStorage.getItem('hacbsShowAppQs');
+    if (hacbsShowAppQs === null) {
+      localStorage.setItem('hacbsShowAppQs', 'false');
+      quickStarts.set('hac-dev', [
```

Comment:
I believe `set` overrides all existing quick starts for the given named app.
Ideally we just add to the existing list. Unsure if there's an API to do so though.

Reply:
so what should we do for this PR? i don't believe there are other quickstarts yet.

Reply:
added a TODO to revisit once we have ability to add new quickstarts to an existing chrome store, and linked the relevant JIRA